components/extension-mgt/org.wso2.carbon.identity.extension.mgt/src/main/java/org/wso2/carbon/identity/extension/mgt/function/JSONObjectToExtensionInfo.java

```diff
@@ -39,6 +39,7 @@ public ExtensionInfo apply(JSONObject jsonObject) {
 
         ExtensionInfo extensionInfo = new ExtensionInfo();
         extensionInfo.setId(jsonObject.getString(ExtensionMgtConstants.ID));
+        extensionInfo.setVersion(jsonObject.optString(ExtensionMgtConstants.VERSION, ""));
```

Comment:
Is setting an empty string okay if the version is not available in the json object?

Reply:
Yes, this is the expectation. Since some users may have previous templates without a version, an empty string will be received through the API in those cases.